dotcms-integration/src/test/java/com/dotmarketing/factories/MultiTreeAPITest.java

```diff
@@ -2913,4 +2904,1251 @@ public void deletePersonalizationSpecificVariant() throws DotDataException {
         assertEquals(1, APILocator.getMultiTreeAPI().getAllContentletReferencesCount(contentlet_4.getIdentifier()));
         assertEquals(1, APILocator.getMultiTreeAPI().getAllContentletReferencesCount(contentlet_5.getIdentifier()));
     }
+
+
+    /**
+     * Method to test: {@link MultiTreeAPIImpl#updateMultiTrees(String, LayoutChanges)}
+     * When: You have a Page with 4 containers all of them are different instances of the same containers,
+     * and you have 5 Contentlets add as follows:
+     * - Contentlet_1 : Add to the instance 1
+     * - Contentlet_2 : Add to the instance 2
+     * - Contentlet_3 : Add to the instance 3
+     * - Contentlet_4 : Add to the instance 3
+     * - Contentlet_5 : Add to the instance 4
+     *
+     * And you move the instance for to be the first one
+     *
+     * Should: The Contentlets should finish as:
+     *
+     * - Contentlet_1 : Add to the instance 2
+     * - Contentlet_2 : Add to the instance 3
+     * - Contentlet_3 : Add to the instance 4
+     * - Contentlet_4 : Add to the instance 5
```

Comment:
shouldn't contentlet 4 be added to instance 4?

Reply:
the assertions doesn't seem to match the test doc

Reply:
It was happened on others test https://github.com/dotCMS/core/pull/28128/commits/1d606b4dc0acc04e7e3cd700cffa93080b2e0c83#diff-5dcfb4ccde2b8bfbbbe18e3a748ae7595e6007af7e248f5c48ee85a808d92f81R2919